docs/io/ascii/fast_ascii_io.rst

```diff
@@ -87,37 +87,31 @@ These parameters are:
 
 .. _fast_conversion_opts:
 
-Parallel and Fast Conversion Options
+Fast Conversion Options
 ------------------------------------
```

Comment:
I like the clean-up. Thanks!

Given Tom A already approved and CI is green, I will just get this suggestion in and merge without running CI again.

```suggestion
Fast Conversion Options
-----------------------
```